envoy/stats/histogram.h

```diff
@@ -161,6 +167,11 @@ class ParentHistogram : public Histogram {
    * Returns the bucket summary representation.
    */
   virtual const std::string bucketSummary() const PURE;
+
+  /**
+   * Returns the bucket summary representation with nonoverlapping buckets.
+   */
+  virtual const std::string nonoverlappingBucketSummary() const PURE;
```

Comment:
`computeNonOverlappingBucketSummary()` 

remove the `const` prefix -- also do this for `bucketSummary()` which shouldn't have the `const` prefix either.

Reply:
Fixed. Removed the `const` prefix from `quantileSummary()` too